blackgate/http_proxy.py

```diff
@@ -2,8 +2,8 @@
 
 import re
 import logging
-from urllib import urlencode
-from urlparse import parse_qs
```

Comment:
The change only works in Python 3. It'll break when running in Python 2. Can we add `try ... except ImportError` or use package `six` to handle it?